### ISSUE TITLE
Microsoft.TestPlatform preview version installed with netcore sdk 2.0 rtm

### ISSUE THREAD

**Issue by livarcocc** [Microsoft/vstest#1134]
_From @evil-shrike on September 1, 2017 9:37_

I installed dotnet SDK 2.0 release.

```
D:\Work\Learn\dotnet\trans_xunit>dotnet --info
.NET Command Line Tools (2.0.0)

Product Information:
 Version:            2.0.0
 Commit SHA-1 hash:  cdcd1928c9

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15063
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.0.0\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
```

But when I run tests with `dotnet test` I can see that Test Execution Command Line Tool Version is of 15.3.0-preview-20170628-02 version:
```
Test run for D:\Work\Learn\dotnet\trans_xunit\bin\Debug\netcoreapp2.0\trans2.dll(.NETCoreApp,Version=v2.0)
Microsoft (R) Test Execution Command Line Tool Version 15.3.0-preview-20170628-02
```

The same version I have when I create a new test project with `dotnet new xunit` or `dotnet new mstest`:
```
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0-preview-20170628-02" />
```

I have VS 2017.3.3 installed. Not preview.

I can see this version in "C:\Program Files\dotnet\sdk\2.0.0\dotnet.deps.json"  as well:
```
  "targets": {
    ".NETCoreApp,Version=v2.0": {
      "dotnet.deps.json/2.0.0-preview3-fnl-006918": {
        "dependencies": {
          "Microsoft.Build.Runtime": "15.3.409",
          "Microsoft.CodeAnalysis.CSharp": "2.3.2-beta1-61921-05",
          "Microsoft.DotNet.TestFramework": "2.0.0-preview3-fnl",
          "Microsoft.TestPlatform.Build": "15.3.0-preview-20170628-02",
..
```

Is it correct? What version of Microsoft.TestPlatform should be used with netcore 2.0?
Can (and should) it be updated separately?

_Copied from original issue: dotnet/cli#7580_

**Comment by livarcocc** [comment]
_From @petarrepac on September 7, 2017 7:38_

Same here

**Comment by livarcocc** [comment]
_From @kuiperzone on September 13, 2017 14:57_

Same here. Driving me mental. Have to "update" every new project to stable package release.

**Comment by livarcocc** [comment]
When the SDK ships, all customer facing components need to move to a final brand. Like 15.3.0. For 15.5, can you make sure that you will insert a final branded version of your package into the CLI?

**Comment by smadala** [comment]
Addressed this for 15.5 in https://github.com/dotnet/cli/pull/7974 . Closing as won't fix for 15.3.
